vortex-ui/src/main/java/mil/army/usace/hec/vortex/ui/GridToPointWizard.java

```diff
@@ -332,6 +351,73 @@ private boolean validateStepTwo() {
     private void submitStepTwo() {}
 
     private JPanel stepThreePanel() {
```

Comment:
You should be able to do this more simply, without the `GridBagConstraints` and all. See `A14TemporalPatternImporterWizard::stepTwoPanel` as an example. Let me know if you have any trouble!